functions_framework_builder/lib/builder.dart

```diff
@@ -65,9 +73,6 @@ class _FunctionsFrameworkBuilder implements Builder {
     );
     await buildStep.writeAsString(mainDart, '''
 // GENERATED CODE - DO NOT MODIFY BY HAND
-// Copyright (c) 2020, the Dart project authors.
```

Comment:
Please don't remove this. I had checked with @grant on this [the other day](https://github.com/GoogleCloudPlatform/functions-framework-dart/pull/4#issuecomment-731823640) about ensuring copyrights even in generated code.

Reply:
ack